ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.Constants.cs

```diff
@@ -252,6 +252,34 @@ public partial class UIManagerModule
         {
             return new Map
             {
+                {
```

Comment:
>{ [](start = 16, length = 1)

Should the drag events be bubbling or direct? We should be consistent across platforms so we should probably try to copy web.

Reply:
These are drop events. Drag events would be a different story. I couldn't find a clear answer on whether they are bubbling in HTML (I guess I'll need to check that wordy W3C spec), but given that an upgrade form direct to bubbling mode is a backward compatible change, I think it's not relevant at this point to mess with bubbling events.